### PR TITLE
Add Open Graph/Twitter meta tags

### PR DESCRIPTION
**Pull request by RoelN**
Note: no content yet.

Fixes https://github.com/undercasetype/fraunces-minisite/issues/51

### CODE REVIEW

**Comment by RoelN** [comment]
@sponcey Would you be so kind to fill https://github.com/undercasetype/fraunces-minisite/blob/5e9c763c7ff9f38564ab189740c5f0c7ac1835cc/src/_data/site.js with the appropriate data? See https://css-tricks.com/essential-meta-tags-social-media/ for details and max. lengths and stuff.

We'd also need a nice image:

> The URL of the image for your object. It should be at least 600x315 pixels, but 1200x630 or larger is preferred (up to 5MB). Stay close to a 1.91:1 aspect ratio to avoid cropping.

Feel free to reach out if you need a hand.

**Comment by sponcey** [comment]
Not sure if I added the image link properly.

**Comment by RoelN** [comment]
@sponcey Thanks! Updated the URLs to serve over `https`.

File: src/_data/site.js [hunk 1]
```diff
@@ -1,5 +1,38 @@
...
+			content: "Fraunces by Undercase Type"
+		},
+		{
+			property: "og:description",
+			content:
+				"Fraunces is a display, 'Old Style' soft-serif typeface inspired by the mannerisms of early 20th century typefaces such as Windsor, Souvenir, and the Cooper Series."
+		},
+		{
+			property: "og:image",
+			content: "https://fraunces.undercase.xyz/img/fraunces_site.png"
```
Comment (pascalw) [inline src/_data/site.js]: @RoelN just a heads-up that this is not going to work, as the image will be processed by webpack so the filename will be mangled. I think we should make the `webpackAsset` function available outside of templates to best resolve this. I'll try to pick it up later today.

**Pull request merged**